### ISSUE TITLE
New database.sql is needed for item table saving its images.

### ISSUE THREAD

**Issue by JacobYim** [JacobYim/CSE442-BuyNSell#132]
New database.sql is needed for item table saving its images.

**Comment by JacobYim** [comment]
The database.sql has been modified like the below 

```
create table user_profile(
    user_id serial primary key,
    fname varchar(24) not null,
    lname varchar(12) not null,
    ubid varchar(12) unique not null,
    email varchar(50) unique not null,
    password varchar(60) not null,
    address1 varchar(60) not null,
    address2 varchar(60),
    city varchar(60),
    zip int not null,
    states varchar(2) not null,
    file_path varchar(60) default null,
    available BOOLEAN not null
);

CREATE TABLE categories(
category_id int unique not null primary key,
parent_id int,
category_name varchar(20) not null
);

insert into categories(category_id, parent_id, category_name)
values (1, null, 'Fashion');

insert into categories(category_id, parent_id, category_name)
values (2, null, 'Electronics');

insert into categories(category_id, parent_id, category_name)
values (3, null, 'Books');

insert into categories(category_id, parent_id, category_name)
values (4, null, 'Home');

insert into categories(category_id, parent_id, category_name)
values (5, null, 'Sports & Outdoor');

insert into categories(category_id, parent_id, category_name)
values (6, null, 'Music');

create table items(
item_id serial primary key,
item_name varchar(50) not null,
time_post timestamp default current_timestamp,
description text,
availability boolean not null default 't',
price int not null,
file_path varchar(60) default null,
post_by int references user_profile(user_id),
item_category int references categories(category_id)
);

create table transaction(
item int references items(item_id),
from_user int references user_profile(user_id),
to_user int references user_profile(user_id),
transaction_date timestamp default current_timestamp
);

--some random input data--
insert into user_profile(fname, lname, ubid, email, password, address1, address2, city, zip, states, file_path, available)
values ('Chulsoo', 'Lim', 'limch', 'limch@buffalo.edu','sha1$f0e604d8$1$d7ae3e858408c66a29af59862a55e94183fc7260','277 American Campus Drive',null,'Amherst',12345,'NY','resource/images/member/limch.jpg','1');

insert into items(item_name,description,price,post_by,item_category)
values ('Laptop Computer','This is pretty clean status',200,1,2);

```

**Comment by JacobYim** [comment]
this change is reflected on the branch item-images

**Comment by JacobYim** [comment]
this is resolved at the merge request  #119
